mne/utils/check.py

```diff
@@ -562,6 +562,40 @@ def _check_depth(depth, kind='depth_mne'):
     return _handle_default(kind, depth)
 
 
+def _check_dict_keys(user_dict, valid_keys,
+                     dict_name="Channel name(s)", valid_name="info"):
+    """Check that the keys in dictionary are valid against a set list.
+
+    Return the input dictionary if it is valid,
+    otherwise raise a ValueError with a readable error message.
+
+    Parameters
```

Comment:
`dict_name` and `valid_name` are missing here. Furthermore, I have trouble associating these names with their function - can we find better names? Or can we not just issue a generic error message, in which case we could drop these parameters?

Reply:
I'm open to alternative suggestions

Reply:
I'd rather keep the more specific names, it makes for better readability at the user end and at least for `_check_option` and `_validate_type` does not seem to add too much burden during development

Reply:
(but I agree the variable names used here could be clearer, like maybe `user_options` and `allowed_options` or something would be clearer)

Reply:
But what does this error mean?

```
Channel name(s) is missing from info: ...
```

Do you mean:

```
Channel name(s) missing in info: ...
```

?

Reply:
I have changed the variable names and error text to suggested values